daemon/cmd/daemon_main.go

```diff
@@ -823,6 +823,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Duration(option.CTMapEntriesTimeoutSYNName, 60*time.Second, "Establishment timeout for entries in TCP CT table")
 	option.BindEnv(vp, option.CTMapEntriesTimeoutSYNName)
 
+	flags.Duration(option.CTMapEntriesTimeoutClosingName, 60*time.Second, "Closing Timeout for entries in TCP CT table")
+	option.BindEnv(vp, option.CTMapEntriesTimeoutClosingName)
+
```

Comment:
Wouldn't it better for backwards compatibility to use the value from `option.CTMapEntriesTimeoutTCPName` as default?